torch/_dynamo/variables/misc.py

```diff
@@ -292,15 +407,23 @@ def call_method(
             assert self.source and not kwargs
             tx.output.side_effects.track_save_for_backward(self, args)
 
-        return variables.ConstantVariable(
-            None, **VariableTracker.propagate(self, args, kwargs.values())
-        )
+        options = VariableTracker.propagate(self, args, kwargs.values())
+        if not hasattr(self, "_saved_tensors"):
+            self._saved_tensors = []
+        for arg in args:
+            # as_proxy can return constant values or other non proxy values
+            if isinstance(arg.as_proxy(), torch.fx.Proxy):
+                arg.as_proxy().node.meta["whitelisted"] = True
+            self._saved_tensors.append(arg)
+        return variables.ConstantVariable(None, **options)
 
     def var_getattr(self, tx, name):
         if name == "save_for_backward":
             return LambdaVariable(
                 lambda *args, **kwargs: self.call_method(tx, name, args, kwargs)
             ).add_options(self)
+        if name == "saved_tensors":
+            return variables.TupleVariable(list(self._saved_tensors))
```

Comment:
nit: should send a tuple to a TupleVariable instead of list?

Reply:
I think it expects `items` to be a list. Grep for `TupleVariable(list` shows the pattern :P

Reply:
Lol wat